botnim/cli_assistant.py

```diff
@@ -44,7 +44,7 @@ def get_assistant_name(assistant_id):
     """
     return client.beta.assistants.retrieve(assistant_id).name
 
-def start_conversation(assistant_id, openapi_spec = None, rtl=False):
+def start_conversation(assistant_id, openapi_spec = None, rtl=False, environment='staging'):
```

Comment:
why not use the default environment here?

Reply:
applied